devito/operations/interpolators.py

```diff
@@ -380,5 +394,6 @@ def interpolation_coeffs(self):
     @property
     def _weights(self):
         ddim, cdim = self.interpolation_coeffs.dimensions[1:]
-        return Mul(*[self.interpolation_coeffs.subs({ddim: ri, cdim: rd-rd.symbolic_min})
+        return Mul(*[self.interpolation_coeffs.subs({ddim: ri,
```

Comment:
nitpicking
for better readability, can we introduce `mapper = {ddim: ....}` on the line above?